src/twisted/names/test/test_client.py

```diff
@@ -24,7 +24,7 @@
 from twisted.names.common import ResolverBase
 
 from twisted.names.test.test_hosts import GoodTempPathMixin
-from twisted.names.test.test_util import MemoryReactor
+from twisted.names.test import test_util
```

Comment:
That's true, but there are *two* `MemoryReactor`s in Twisted.  This makes it clear which one the test uses.